cilium-cli/connectivity/builder/north_south_loadbalancing.go

```diff
@@ -16,5 +16,8 @@ func (t northSouthLoadbalancing) build(ct *check.ConnectivityTest, _ map[string]
 		WithFeatureRequirements(
 			withKPRReqForMultiCluster(ct, features.RequireEnabled(features.NodeWithoutCilium))...,
 		).
-		WithScenarios(tests.OutsideToNodePort())
+		WithScenarios(
+			tests.OutsideToNodePort(),
+			tests.OutsideToNodePortForHostNetNs(),
+		)
```

Comment:
For compatibility we'll need some variant of `WithCiliumVersion(">=1.18.0")` here, until the backports to older Cilium versions have landed.